docs/docs/additional/array.mdx

```diff
@@ -90,6 +90,34 @@ articlesStore.update(1, ({ comments }) => ({
 }));
 ```
 
+### `arrayToggle`
+
+Adds a value to an array if it does'nt exist yet or removes it if already present. Objects are compared by identity by
```

Comment:
Adds a value to an array if it doesn't exist yet or removes it if already present. Objects are compared by identity by default. You can override it by providing a compare function.